README.md

```diff
@@ -104,6 +105,7 @@ public function getCacheableProperties(): array {
         'prefix' => 'cacheable',
         'identifier' => 'id',
         'logging' => [
+            'channel' => 'default',
```

Comment:
@Sleepy4k please choose another example name here, as "default" could be misunderstood. Maybe `anotherChannel`. Thanks